evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -1407,3 +1409,30 @@ def test_partial_dependence_datetime_extra(
         assert len(part_dep["partial_dependence"]) == 10
         assert len(part_dep["feature_values"]) == 10
     assert not part_dep.isnull().any(axis=None)
+
+
+@pytest.mark.parametrize(
+    "cols,expected_cols", [(0, [0]), ([0, 1], [0, 1]), ([0, 2], [0]), (2, [])]
+)
+@pytest.mark.parametrize("types", ["URL", "EmailAddress", "NaturalLanguage"])
+def test_partial_dependence_not_allowed_types(types, cols, expected_cols, X_y_binary):
+    # test to see if we can get partial dependence fine with a dataset that has unknown features
+    X, y = X_y_binary
+    X = pd.DataFrame(X)
+    X.ww.init(logical_types={0: types, 1: "URL"})
```

Comment:
Is the intent here for "URL" to be the parametrized type?

Reply:
@chukarsten This makes col 0 parametrized with the passed-in types, and col 1 be of type 'URL'. This allows me to make sure if I pass in 2 non-allowed columns, (0, 1) to partial dependence, I get the message that both columns aren't allowed and that both types are reported.

Reply:
I see now, I totally misread this.